### PR TITLE
Add 20 image seeds

### LINKED ISSUE #5: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

__Story__:

Working on this application has been a lot of fun, right? You are fast
becoming a git expert, and an expert at working with a deployment
pipeline. Speaking of development and deployment pipelines, is it not pretty
annoying that everytime a new deployment is created it starts out with an empty
database? It seems like everytime a review application is created we need to
manually add a handful of images in order to manually test the
application. Would it not be great if each deployment was created with a
handful of existing images?

__Acceptance criteria__:
- [x] After running `rake db:setup` at least 20 unique images are shown in the
  homepage.
- [x] When a new app is deployed on heroku (e.g., every review app), that app
  automatically has at least 20 unique images on its homepage.

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by jschneid**
Closes #5

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -4,4 +4,29 @@
...
+  https://image.slidesharecdn.com/201509makingthemostofappfolioforblogpost-151001041802-lva1-app6891/95/making-the-most-of-appfolio-51-638.jpg?cb=1443673230
+  https://assets.themuse.com/uploaded/companies/810/f1.jpg?v=2d5dbf23015bb423e0441092a714318b275d1aa172db4746b7efa7d14a7bbe0d
+  https://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Appfolio_HQ.jpg/1200px-Appfolio_HQ.jpg
+  http://is3.mzstatic.com/image/thumb/Purple118/v4/96/09/f1/9609f13c-0b13-8a79-6c98-645790aef9d3/source/175x175bb.jpg
+  https://blog.appfolio.com/wp-content/uploads/2015/10/customer-conference-2015b-300x200.jpg
+  https://blog.appfolio.com/wp-content/uploads/2014/02/appfolio_a_heart-e1394234410671.png
+  http://blog.appfolio.com/wp-content/uploads/2015/04/Rentlinx1.jpg
+  https://static.wixstatic.com/media/c0ff70_ce18a0c20fbc45048ef881d6e407bc6c~mv2.jpg
+].each do |url|
+  Image.create!(url: url)
```
Comment (mlewisno) [inline db/seeds.rb]: Great to see you using `create!` - people often miss that that is needed if you want to know then the call fails.

My only feedback is that there's a way you can create these 20 images without calling each - are you familiar with it? I don't think it's required in this PR but it's good to know about!

File: db/seeds.rb [hunk 2]
```diff
@@ -4,4 +4,29 @@
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+#   Character.create(name: 'Luke movie: movies.first)
```
Comment (mlewisno) [inline db/seeds.rb]: Need these comments still? ;)

File: db/seeds.rb (hunk 1 above)
Comment (jschneid) [reply db/seeds.rb]: Being brand new to Ruby/Rails, the only other approach that I'm aware of was to call `create` on every single URL string individually.

So yes, I'm eager to learn about alternatives and get more tools added to my toolbox! What's the approach that you're referring to?

File: db/seeds.rb (hunk 2 above)
Comment (jschneid) [reply db/seeds.rb]: Good point, might as well get rid of the boilerplate comments! Fixed them up in 4d7f5fe0edaf87e7f5d89fc7903f156aea010705.

File: db/seeds.rb (hunk 1 above)
Comment (mlewisno) [reply db/seeds.rb]: Sorry for the delay! We had a big team meeting most of the day! Yeah, basically you can pass a single array of hashes into a call to `create!` and it'll create one object per hash, passing in the hashes as the parameters for each creation. So, `Images.create!([{ url: blah },{ url: blah2 }])` creates two Images, one with url of `blah` and one with `blah2`

**Pull request merged**